woodpecker-go/woodpecker/interface.go

```diff
@@ -116,7 +116,7 @@ type Client interface {
 	Deploy(repoID, pipeline int64, env string, params map[string]string) (*Pipeline, error)
 
 	// LogsPurge purges the pipeline logs for the specified pipeline.
-	LogsPurge(repoID, pipeline int64) error
+	LogsPurge(repoID, pipeline int64, step int64) error
```

Comment:
is breaking ...

you could make step optional via:
```suggestion
	LogsPurge(repoID, pipeline int64, step ...int64) error
```

Reply:
Thanks, that is what I meant.